tfexec/version_test.go

```diff
@@ -300,16 +300,16 @@ func TestExperimentsEnabled(t *testing.T) {
 		tfVersion     *version.Version
 		expectedError error
 	}{
-		"experiments-enabled-in-1.9.0-alpha20240404": {
-			tfVersion: version.Must(version.NewVersion(testutil.Alpha_v1_9)),
+		fmt.Sprintf("experiments-enabled-in-%s", testutil.Latest_Alpha_v1_9): {
+			tfVersion: version.Must(version.NewVersion(testutil.Latest_Alpha_v1_9)),
 		},
-		"experiments-disabled-in-1.8.0-beta1": {
-			tfVersion:     version.Must(version.NewVersion(testutil.Beta_v1_8)),
-			expectedError: errors.New("experiments are not enabled in version 1.8.0-beta1, as it's not an alpha or dev build"),
+		fmt.Sprintf("experiments-disabled-in-%s", testutil.Latest_Beta_v1_8): {
```

Comment:
```suggestion
		"experiments-disabled-in-betas": {
```

I think removing the exact version from the map key will make it more readable